pennylane/ops/op_math/decompositions/normal_forms.py

```diff
@@ -20,12 +20,6 @@
 import pennylane as qml
 from pennylane.ops.op_math.decompositions.rings import _SQRT2, DyadicMatrix, SO3Matrix, ZOmega
 
-is_jax = True
```

Comment:
Why are we shifting this inside the function?

Reply:
Do you think re-import will cause a performance hit in the subsequent function calls?